github-dark.user.css

```diff
@@ -9,7 +9,6 @@
 @-moz-document domain("github.com") {
```

Comment:
Could you pull the formatting changes out into a second PR? Also it seems your formatter is removing newlines where I put them so if you could leave those in place that would be good.
You also just copied and pasted the `.user.css` into the `.css`. Just don't worry about the `.css` file, I'm going to deprecate that soon.
Could you also provide some screenshots so I can verify the changes on my end too? Thanks!